app/views/taxons/confirm_delete.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1>You are about to delete "<%= taxon.internal_name %>"</h1>
+<h1>You are about to delete "<%= taxon.title %>"</h1>
```

Comment:
Hasty code changes in the related controller. Just seen we have access to a proper taxon object with internal name on it. Updated.